feature-libs/product-configurator/rulebased/components/attribute/quantity/configurator-attribute-quantity.service.ts

```diff
@@ -11,14 +11,16 @@ import { Configurator } from '../../../core/model/configurator.model';
   providedIn: 'root',
 })
 export class ConfiguratorAttributeQuantityService {
+  readonly RETRACT_VALUE_CODE = Configurator.RetractValueCode;
```

Comment:
Why is the needed, why not directly referring to Configurator.RetractValueCode in disableQuantityActions?